packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -257,6 +261,8 @@ contract GenArt721CoreV3_Engine is
     // royalty split provider
     ISplitProviderV0 public splitProvider;
 
+    constructor() Ownable(msg.sender) {}
```

Comment:
nit - I think continuing to use the non-upgradeable ownable is the right move, and we just transfer to AdminACL in initializer like you are doing. Because this constructor is only ever ran on the implementation contract, I would recommend heavily commenting on why this is only required to satisfy the constructor requirements of the Ownable abstract contract, it not actually used. I think setting owner as `0xdead` or something other than `msg.sender` would help make that clear too?

Reply:
Thanks for catching this!! Added a comment here and in flex + updated to 0xdead 🙏 :rip: